test/rpc/test_rpc_misc.py

```diff
@@ -123,17 +97,3 @@ def test_chain_id(deploy_info):
 
     assert isinstance(rpc_chain_id, str)
     assert rpc_chain_id == hex(chain_id)
-
-
-def test_protocol_version(deploy_info):
```

Comment:
I think it's okay not to support it and add a note in README about it not being supported.

The method will be removed since the configuration of it is unclear, and the use case for using it is unclear.